sdk/communication/Azure.Communication.CallingServer/src/CallConnection.cs

```diff
@@ -22,11 +24,38 @@ public class CallConnection
         /// </summary>
         public virtual string CallConnectionId { get; internal set; }
 
-        internal CallConnection(string callConnectionId, CallConnectionsRestClient callConnectionRestClient, ClientDiagnostics clientDiagnostics)
+        /// <summary>
+        /// The ServerCall id.
+        /// </summary>
+        public virtual string ServerCallId { get; internal set; }
+
+        /// <summary> The source of the call. </summary>
+        public virtual CommunicationIdentifier Source { get; internal set; }
```

Comment:
Should Source and AlternateCallerId be wrapped into a CallSource object here?

Reply:
Updated